hasher-matcher-actioner/hmalib/lambdas/api/lcc.py

```diff
@@ -0,0 +1,65 @@
+from dataclasses import dataclass, asdict
+from mypy_boto3_dynamodb.service_resource import Table
+import bottle
+
+from hmalib.common.config import HMAConfig
+from hmalib.common.models.bank import BankMember, BanksTable
+
+from hmalib.lambdas.api.middleware import (
+    jsoninator,
+    JSONifiable,
+    DictParseable,
+    SubApp,
+)
+
+import typing as t
+
+
+# I return this type because I saw in the documentation that a
+# to_json method is required
+
+# I might need to use
+# `jsoninator(<requestObj>, from_query=True)` but not sure how the from_query=True
+# would modify this implementation
+@dataclass
+class LccResponse(JSONifiable):
```

Comment:
nit: PEP8 recommends using LCCResponse in this case. https://legacy.python.org/dev/peps/pep-0008/#descriptive-naming-styles